.github/workflows/ci-cleanup-old-workflow-runs.yml

```diff
@@ -0,0 +1,16 @@
+---
+name: "CI Cleanup workflow runs"
+
+on:
+  schedule:
+    - cron: '42 6 * * 0'
+
+jobs:
+  ci-workflow-cleanup:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: dmvict/clean-workflow-runs@9b0fba98c68369add0884d6af67e6866578cbf98 # pin@v1.0.0
+        with:
+          token: ${{ secrets.PRIVATE_GITHUB_TOKEN }}
+          save_period: 30
```

Comment:
I like the idea of the PR but was just wondering whether `save_period: 30` might be overly restrictive. Perhaps there are some situations when one would like to look back at runs from over a month ago? I'm not super against 30, but I would feel more comfortable with, say, 60 or 90.

Reply:
Do you have any concrete example, where a longer history was actually necessary?
I would rather go for the 30 days now and only adopt this to a longer period, on the occasion of a specific need rather than on a vague feeling.

Reply:
My feeling was that it was useful to look back at what the state of the CI was before it broke recently, and in particular it took over 30 days to fix.